plugins/generator-1.21.1/neoforge-1.21.1/procedures/world_send_chat.java.ftl

```diff
@@ -1,2 +1,13 @@
-if (!world.isClientSide() && world.getServer() != null)
-	world.getServer().getPlayerList().broadcastSystemMessage(Component.literal(${input$text}), false);
\ No newline at end of file
+if (world instanceof ServerLevel _level) {
+	MutableComponent message = Component.literal(${input$text!"Message"});

```

Comment:
Since this all code is part of the same system, you should leave out new line spacing here and below too, applies to both versions